### ISSUE TITLE
The latest April Fools Update (24w14potato) doesn't run with Fabric API though it works in the official Minecraft Launcher.

### ISSUE THREAD

**Issue by Mizuuta** [PrismLauncher/PrismLauncher#2267]
### Operating System

Linux

### Version of Prism Launcher

Prism Launcher 9.0-develop (Commit: b7835036d4b4f392ea97cc6b021c152f00cd8e69)

### Version of Qt

Qt 6.2.4

### Description of bug

If I try to launch the 24w14potato April Fools Snapshot with just Fabric API installed Minecraft crashes although in works without a problem in the official Minecraft Launcher.

### Steps to reproduce

1. Make a new Instance with 24w14potato and Fabric 0.15.9
2. Edit the Instance and go to the "Mods" tab
3. Download Fabric API version "[24w14potato] Fabric API 0.96.14+24w14potato [beta]"
4. Launch the game and see how it crashes

### Suspected cause

It probably happened because 24w14potato got reuploaded to fix some bugs and Fabric API is for that version. So Prism Launcher probably still downloads the old 24w14potato version. 

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by maskersss** [comment]
works fine for me, https://mclo.gs/fWg0qeS
a log of it crashing could be helpful

i also downloaded 24w14p for the first time way after it was reuploaded, so it could maybe the case that you downloaded the first version initially and prism just didn't update it?

**Comment by Mizuuta** [comment]
Fixed it by first moving the ".local/PrismLauncher" folder to a diffrent place as Backup. Then I restarted Prism Launcher, configured everything and tried launching 24w14p with Fabric API again and it worked. Then I tried to move the new "PrismLauncher" folder somewhere else as Backup, restored the old one and just try to copy the assets and libraries from the new one to the old one which didn't work because Prism downloaded the old 24w14p version again instead of just using the new one (Maybe could've made a new instance with same stuff to see if it works). I ended up with just using the new one and moving the instnces I need to it which then all worked.
